framework/JobHandler.py

```diff
@@ -401,9 +418,20 @@ def isFinished(self):
       @ In, None
       @ Out, isFinished, bool, True all the runs in the queue are finished
     """
+
+    '''
+    FIXME: The following two lines of codes have been a temporary fix for timing issues
+           on the collections of jobs in the jobHandler. This issue has emerged when
+           performing batching. It is needed to review the relations between jobHandler
+           and the Step when retrieving multiple jobs.
+           An issue has been opened: 'JobHandler and Batching #1402'
+    '''
+    #import time
+    #time.sleep(0.001)
```

Comment:
Do you still want to keep these lines of comments?

Reply:
removed